tests/linearGaussian_npse_test.py

```diff
@@ -223,11 +223,9 @@ def test_npse_map():
     theta = prior.sample((num_simulations,))
     x = linear_gaussian(theta, likelihood_shift, likelihood_cov)
 
-    inference.append_simulations(theta, x).train(
-        training_batch_size=100, max_num_epochs=10
-    )
+    inference.append_simulations(theta, x).train()
     posterior = inference.build_posterior().set_default_x(x_o)
 
-    map_ = posterior.map(show_progress_bars=True)
+    map_ = posterior.map(show_progress_bars=True, num_iter=5)
 
-    assert torch.allclose(map_, gt_posterior.mean, atol=0.4), "MAP is not close to GT."
+    assert torch.allclose(map_, gt_posterior.mean, atol=0.15), "MAP is not close to GT."
```

Comment:
Great that we can now reduce the tolerance. Actually, I had a quick look at how we test the MAP for the other methods, and we actually make a different assertion, e.g. for `linearGaussian_snpe_test`, we assert
`assert ((map_ - gt_posterior.mean) ** 2).sum() < 0.5`

Maybe it makes sense to make it the same test across methods, although also happy for this to be done in a different PR

Reply:
good catch, will update it.

Reply:
map test passes.